README.md

```diff
@@ -42,11 +43,38 @@ to execute the unit tests:
 
 ```sh
 make test # run all tests
-cd src && cabal v2-test # run haskell tests
+cabal v2-test # run haskell tests
 ```
 
 To update the project dependencies run:
 
 ```sh
 nix flake update
 ```
+
+# Usage
+
+Once you are in the nix shell, you can use act backends as follows.
+
+## SMT
+
+```sh
+act prove --file <PATH_TO_SPEC>
```

Comment:
Being able to use the `act` executable relies on having run `make` first, as it is the Makefile that runs the command to build it (of course one could also run it by hand).

I suggest using `cabal run act` instead which has no prerequisites, or specifying that `make` has to be run first.